app/models/solidus_subscriptions/subscription.rb

```diff
@@ -228,5 +230,10 @@ def check_total_skips_exceeded
     def line_item
       line_items.first
     end
+
+    def create_payment_method_profile
+      payment = OpenStruct.new(source: self.wallet_payment_source.payment_source)
+      payment.source.payment_method.create_profile(payment)
```

Comment:
Is this code seen somewhere else, maybe in `solidus_paypay_braintree`?

Reply:
This is from solidus in a payment `after_save` https://github.com/geminimvp/solidus/blob/master/core/app/models/spree/payment.rb#L234

Reply:
OK, can you verify that this fills in the `token` attribute that we need? This sounds tough to put in a spec.

Reply:
Here is a source that was updated from subscription update with the token included:

![image](https://user-images.githubusercontent.com/10490055/51133892-b1e02e00-17fb-11e9-8623-683ae7241a34.png)